<launch>
  <param name="vu_r" value="2.45"/>
  <param name="vu_l" value="2.0" />
  <param name="vu_m" value="0.9"/>
  <param name="max_steer_left" value="10" />
  <param name="max_steer_right" value="170"/>
  <param name="mid_steer_right" value="170"/>
  <param name="kp_park" value="2.5"/>
  <!-- <param name="hough_thr" value="30  "/>
  <param name="min_lin_len" value="20" />
  <param name="max_gap_len" value="20" />
  <param name="value_thr_low" value="0"/>
  <param name="value_thr_high" value="100" />
  <param name="canny_thr_low" value="50" />
  <param name="canny_thr_high" value="100" />
  <param name="calib_file" value="/root/catkin_ws_user/Matrix.yaml" /> -->

 <node pkg="laser_scan" type="laser_scan" name="laser_scan" output="log">
     </node>

 <!-- <node pkg="lane_detection" type="lane_hough_node" name="lane_hough_node" output="screen">
   <remap from="image" to="/app/camera/rgb/image_raw"/> -->

   <node pkg="navigation_toretto" type="park_toretto" name="park_toretto" output="screen">
     </node>

</launch>
